AndroidNativeLoginTemplate/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+	android:versionCode="1"
+	android:versionName="1.0"
+	android:installLocation="internalOnly">
+
+	<application android:icon="@drawable/sf__icon"
+	    android:label="@string/app_name"
+		android:name=".MainApplication"
+		android:manageSpaceActivity="com.salesforce.androidsdk.ui.ManageSpaceActivity">
+
+		<!-- Launcher screen -->
+		<activity android:name=".MainActivity"
+			android:theme="@style/SalesforceSDK"
+            android:exported="true">
+			<intent-filter>
+				<action android:name="android.intent.action.MAIN" />
+				<category android:name="android.intent.category.LAUNCHER" />
+			</intent-filter>
+		</activity>
+
+		<activity android:name=".NativeLogin"
+			android:theme="@style/NativeLoginTheme"
+			android:windowSoftInputMode="adjustResize"
+			android:exported="false" />
```

Comment:
`adjustResize` is necessary for having input fields not be hidden behind the keyboard.